Goldleaf/Program/RomFs/Language/Strings/de.json

```diff
@@ -225,36 +225,36 @@
     "Soll der Inhalt aus der Zwischenablage hier eingefügt werden?",
     "Es wurde nichts zum Kopieren ausgewählt.",
     "Wähle einen Namen um eine neue Datei zu erstellen",
-    "Eine andere Datei oder Ordner existiert bereits mit dem Namen.",
+    "Eine andere Datei oder Ordner existiert bereits mit diesem Namen.",
     "Datei wurde erstellt:",
     "Ein Ordner wurde erstellt:",
     "Neustart oder Ausschalten",
     "Soll die Konsole neustarten oder ausgeschaltet werden?",
-    "Datei oder Ordner wurde kopiert. Die Zwischenable wurde aktualsiert.",
+    "Datei oder Ordner wurde kopiert. Die Zwischenablage wurde aktualsiert.",
     "Neustart",
     "Ausschalten",
     "Ok",
     "Titel Key;",
     "Signatur Typ:",
     "Ordnergröße:",
-    "Wähle einen neuen Namen um die umzubenennen.",
```

Comment:
It should be:
"Wähle einen neuen Namen." or Wähle einen neuen Namen um die Datei umzubenennen."